montecarlo/cfast_mcarlo.py

```diff
@@ -231,7 +237,7 @@ def _section_doors_and_holes(self):# {{{
             vent_to_name = v['vent_to_name'].split(".")[0]
```

Comment:
splitting "." is already done here

Reply:
Ok, I will unify processing vent_to_name and vent_from_name across this class. Thanks!

Reply:
fixed